sidebars/pipelines.js

```diff
@@ -44,10 +64,33 @@ const sidebar = [
         ],
       },
       {
-        label: "Maintain Pipelines",
+        label: "Configuring Pipelines",
         type: "category",
         collapsible: false,
-        items: ["pipelines/maintain/updating", "pipelines/maintain/extending"],
+        items: [
+          {
+            label: "Access Control",
+            type: "doc",
+            id: "pipelines/access-control/index",
+          },
+          {
+            label: "Pipelines Actions",
+            type: "doc",
+            id: "pipelines/pipelines-actions/index",
+          },
+        ],
+      },
+      {
+        label: "Data & privacy",
+        type: "category",
+        collapsible: false,
+        items: [
+          {
+            label: "Usage Data",
+            type: "doc",
+            id: "pipelines/usage-data/index",
+          },
+        ],
       },
       {
         label: "Previous Versions",
```

Comment:
Could we somehow include the "external link" icon like we have on Knowledge Base to make clear the user will jump to a different section here? Alternatively, we could make this a "standard" page and the link out to the ECS Deploy Runner pipelines from there?

<img width="158" alt="image" src="https://github.com/gruntwork-io/docs/assets/4295964/cb4e086f-2954-40e5-9512-d2901ff340f2">